Xamarin.Forms.Core/DatePicker.cs

```diff
@@ -56,6 +62,44 @@ public Color TextColor
 			set { SetValue(TextElement.TextColorProperty, value); }
 		}
 
+		public FontAttributes FontAttributes
+		{
+			get { return (FontAttributes)GetValue(FontAttributesProperty); }
+			set { SetValue(FontAttributesProperty, value); }
+		}
+
+		public string FontFamily
+		{
+			get { return (string)GetValue(FontFamilyProperty); }
+			set { SetValue(FontFamilyProperty, value); }
+		}
+
+		[TypeConverter(typeof(FontSizeConverter))]
+		public double FontSize
+		{
+			get { return (double)GetValue(FontSizeProperty); }
+			set { SetValue(FontSizeProperty, value); }
+		}
+
+		void IFontElement.OnFontFamilyChanged(string oldValue, string newValue)
+		{
+		}
```

Comment:
any reason why we don't InvalidateMeasure on font change ? Do we have tests showing that it's not required on any platform ?